rules/windows/process_creation/proc_creation_win_remote_access_tools_meshagent_arguments.yml

```
@@ -0,0 +1,35 @@
+title: Remote Access Tool - Potential MeshAgent Usage - Windows
+id: 2fbbe9ff-0afc-470b-bdc0-592198339968
+related:
+    - id: 22c45af6-f590-4d44-bab3-b5b2d2a2b6d9
+      type: similar
+status: experimental
+description: |
+    Detects potential usage of MeshAgent which is a tool used for remote access.
+    Historical data shows that threat actors rename MeshAgent binary to evade detection.
+    Matching command lines with the '--meshServiceName' argument can indicate that the MeshAgent is being used for remote access.
+references:
+    - https://www.huntress.com/blog/know-thy-enemy-a-novel-november-case-on-persistent-remote-access
+    - https://thecyberexpress.com/ukraine-hit-by-meshagent-malware-campaign/
+    - https://wazuh.com/blog/how-to-detect-meshagent-with-wazuh/
+    - https://www.security.com/threat-intelligence/medusa-ransomware-attacks
+author: Norbert Jaśniewicz (AlphaSOC)
+date: 2025-05-19
+modified: 2025-05-19
+tags:
+    - attack.command-and-control
+    - attack.t1219
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection:
+        CommandLine|windash|contains: --meshServiceName
```

```suggestion
        CommandLine|contains|windash: --meshServiceName
```
to keep the logic with the rest of the repo

---

Isn't windash meant to be used for a single '-' or '/'?
if the intention to use windash is to also watch for '/meshServiceName', the windash won't do it because here is the pysigma conversion of this rule to SPL
``
CommandLine="*--meshServiceName*" OR CommandLine="*-/meshServiceName*" OR CommandLine="*-–meshServiceName*" OR CommandLine="*-—meshServiceName*" OR CommandLine="*-―meshServiceName*" | table CommandLine,ParentImage,Image
``

I'm also not sure if /meshServiceName works instead of `--meshServiceName`

---

You're right, it's not a use case for windash.
I've verified that neither `/meshServiceName` nor `-meshServiceName` does not work instead of `--meshServiceName` so it can be just removed.